benchmarks/forms.py

```diff
@@ -36,19 +36,6 @@ class UploadPlaceHolder(forms.Form):
 
 
 class UploadFileForm(forms.Form):
-    model_type = forms.ChoiceField(choices=[
-        ("BaseModel", "Base model - to submit a standard machine learning model"),
-        ("BrainModel", "Brain model - to change brain-transformation")])
-    zip_file = forms.FileField(label="", help_text='Required')
-    public = forms.BooleanField(label='Make model scores public (can be changed later):', required=False,
-                                help_text='Check if you want the results of your submitted models included in the public ranking.')
```

Comment:
I'm confused why we want to remove this; this has nothing to do with whether or not the code/weights are public. @kvfairchild I feel like there's a breakdown in communication somewhere